src/panel/direction/PublicTransportLine.jsx

```diff
@@ -20,15 +21,20 @@ const PublicTransportLine = ({ mode, info }) => {
     }
   }
   const lineColor = info.lineColor ? Color('#' + info.lineColor) : Color('white');
-  return <span
-    className={classnames('routePtLine', { 'routePtLine--dark': lineColor.isDark() })}
-    style={{
-      backgroundColor: lineColor.hex(),
-      borderColor: lineColor.rgbNumber() === 0xffffff ? 'black' : lineColor.hex(),
-    }}
-  >
-    {type} {info.num}
-  </span>;
+  return <>
+    <span
+      className={classnames('routePtLine', { 'routePtLine--dark': lineColor.isDark() })}
+      style={{
+        backgroundColor: lineColor.hex(),
+        borderColor: lineColor.rgbNumber() === 0xffffff ? 'black' : lineColor.hex(),
+      }}
+    >
+      {type} {info.num}
+    </span>
+    {showDirection && <span className={classnames('routePtLine-direction', 'u-text--subtitle')}>
+      {_('Towards', 'direction')} {info.direction}
```

Comment:
I would recommend to use a formatted string here, as other languages may not use a simple prefix.  
See https://github.com/Qwant/erdapfel/blob/master/src/panel/direction/WalkLeg.jsx#L11-L13 for example.

Reply:
of course, my bad. fixed!